### ISSUE TITLE
Not support Cyrillic symbols

### ISSUE THREAD

**Issue by agorshkov23** [zeroturnaround/sql-formatter#46]
Example:
Original:
```sql
SELECT test, тест FROM table;
```
Result:
```sql
SELECT
  test,
  т е с т
FROM
  table;
```

**Comment by agorshkov23** [comment]
Fast hack: modify regex
```js
	    Tokenizer.prototype.createWordRegex = function createWordRegex() {
	        var specialChars = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : [];

	        return new RegExp("^([\\w" + specialChars.join("") + "а-яА-Я]+)");
	    };
```

**Comment by mtxr** [comment]
@zeroturnaround any updates on this?

**Comment by kufii** [comment]
I fixed this issue in PR #86. While we wait for it to be merged you can use my fork that I published with this fix. https://www.npmjs.com/package/sql-formatter-plus

**Comment by nene** [comment]
Fixed now in 3.1.0 release.
